### ISSUE TITLE
Unable to resolve dependency tree

### ISSUE THREAD

**Issue by saurav2799** [react-materialize/react-materialize#1196]
### Description
Install error in new react app created with create-react-app

### Steps to Reproduce

1. setup project: npx create-react-app .
2. install package: npm i react-materialize
3. [or Code Sample...]

**Expected behavior:**
added xx packages, and audited xx packages in xxm

**Actual behavior:**

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: xxxxxx
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.12.0" from react-materialize@3.9.8
npm ERR! node_modules/react-materialize
npm ERR!   react-materialize@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Versions

**react-materialize:** 3.9.8
**react:** 17.0.2
**npm:** 7.14.0

**Comment by tagplus5** [comment]
Try to `npm install react-materialize --legacy-peer-deps`

**Comment by NatyveCoder** [comment]
@tagplus5 is this safe?

**Comment by tagplus5** [comment]
> @tagplus5 is this safe?

https://stackoverflow.com/a/66620869/4054751

```Adding --legacy-peer-deps to your npm installation will bypass peerDependency auto-installation, but this may result in conflicts due to potentially breaking changes```

**Comment by alextrastero** [comment]
Should be solved in `react-materialize@3.10.0`
